src/pages/components/toggle/style.mdx

```diff
@@ -6,6 +6,23 @@ description:
 tabs: ['Usage', 'Style', 'Code', 'Accessibility']
 ---
 
+<PageDescription>
+
+The following page documents visual specifications such as color, typography,
+structure, and size.
+
+</PageDescription>
+
+<AnchorLinks>
+
+<AnchorLink>Color</AnchorLink>
+<AnchorLink>Typography</AnchorLink>
+<AnchorLink>Structure</AnchorLink>
+<AnchorLink>Sizes</AnchorLink>
```

Comment:
According to the latest Style tab template, this should be "Size," not "Sizes.", this should be "Size," not "Sizes."

```suggestion
<AnchorLink>Size</AnchorLink>
```